clueless/models.py

```diff
@@ -1,13 +1,163 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-class Room(models.Model):
-	title = models.CharField(max_length=30)
-	background = models.CharField(max_length=50)
-	position = models.IntegerField()
+class Board(models.Model):
+    """
+    Board object for the entire game.  Should be referenced by multiple games and space collections
+    """
+    pass
 
+class SpaceCollection(models.Model):
+    """
+    Any implementations should occupy a set of spaces on the board
+    """
+    board = models.ForeignKey(Board)
+
+class Space(models.Model):
+    """
+    Represents a space on the game board, with an X Y position and all neighbor spaces
+    """
+    posX = models.IntegerField()
+    posY = models.IntegerField()
+    spaceNorth = models.OneToOneField('self', related_name='spaceSouth')
+    spaceEast = models.OneToOneField('self', related_name='spaceWest')
+    spaceCollector = models.ForeignKey(SpaceCollection)
+
+class Player(models.Model):
+    """
+    Represents a player in the context of a clueless game.  Ties back to Django user
+    """
+    user = models.ForeignKey(User)
+    currentSpace = models.ForeignKey(Space)
+
+class Hallway(SpaceCollection):
+    """
+    Hallway in the game
+    """
+    pass
+
+class SecretPassage(SpaceCollection):
+    """
+    Secret Passage in the game
+    """
+    pass
+
+class Card(models.Model):
+    """
+    A game Card.  Can either be a Room, Character, or Weapon
+    """
+    card_id = models.AutoField(primary_key=True) #had to override, due to multiple inheritence conflicts later
+    name = models.CharField(max_length=30)
+
```

Comment:
Hmm wouldn't a value be the attribute though? I feel like we need to be saving that somewhere even if not in here